src/dos/dos_classes.cpp

```diff
@@ -308,6 +309,119 @@ bool DOS_PSP::SetNumFiles(uint16_t file_num)
 	return true;
 }
 
+static constexpr auto bytes_to_read = 1024;
```

Comment:
Why 1024? Is this some hard limit for our DOS, enforced in other place? If so, it would be nice to have some comment. If I remember correctly, MS-DOS had a limit of 127 characters, or so.

If there is no real limit, maybe it would be better to have some dynamic buffer?

Reply:
> Why 1024? Is this some hard limit for our DOS, enforced in other place? If so, it would be nice to have some comment. If I remember correctly, MS-DOS had a limit of 127 characters, or so.

MS-DOS's environment space can be dialed up to 1024 characters with the `/e` (environment) flag at boot-time (I think in `CONFIG.SYS`):

```
SHELL=COMMAND.COM /e:1024
```

Maybe that's the connection here?

Reply:
`BytesToRead` is the style for constants.